Assets/Mirror/Components/NetworkTransformBase.cs

```diff
@@ -312,7 +312,10 @@ bool HasMovedOrRotated()
         void ApplyPositionAndRotation(Vector3 position, Quaternion rotation)
         {
             targetComponent.transform.position = position;
-            targetComponent.transform.rotation = rotation;
+            if (Compression.NoRotation != compressRotation)
```

Comment:
I like it,  but can we also avoid serializing the rotation?

Reply:
So should I make something like an ApplyPosition so I can leave out passing around a rotation?

Reply:
no,  this is perfectly good.   But there is another part of this class where we serialize the rotation and we deserialize the rotation.    Would be good to skip the rotation if we are using NoRotation

Reply:
SerializeIntoWriter already looks like it won't write the rotation if you set it to NoRotation. Unless it's something else I'm missing.

Reply:
you are right,  this is good as is.  Thanks.